src/index.tsx

```diff
@@ -33,23 +34,32 @@ function createCanvasInstance<Props extends SketchProps = SketchProps>(
 }
 
 function removeCanvasInstance<Props extends SketchProps = SketchProps>(
-  canvasInstanceRef: MutableRefObject<P5CanvasInstance<Props> | undefined>
+  canvasInstanceRef: React.MutableRefObject<P5CanvasInstance<Props> | undefined>
 ) {
   canvasInstanceRef.current?.remove();
   canvasInstanceRef.current = undefined;
 }
 
-function ReactP5WrapperComponent<Props extends SketchProps = SketchProps>({
+function ReactP5WrapperComponentGuard<Props extends SketchProps = SketchProps>({
```

Comment:
Add a guard to catch runtime validations of input data, in this case the sketch is what we are looking to validate.